Application/Validation/NameValidator.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Core.Dto;
+using Core.Dto.Request;
+using FluentValidation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Application.Validation
+{
+    public class NameValidator : AbstractValidator<NameDto>
+    {
+        public NameValidator(GeoLocationValidator geoLocationValidator)
```

Comment:
For consistency, you could consider injecting all the validators instead of only `GeoLocationValidator`.

Reply:
Done